lib/mi_bridges/driver/vehicles_page.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class VehiclesPage < BasePage
+      TITLE = "Vehicles"
+
+      def setup; end
+
+      def fill_in_required_fields
+        click_id "assetOther_1"
```

Comment:
maybe we can alias this into a private method so it's clearer what we are clicking on?